src/test/java/com/autotune/analyzer/utils/TestEvalExParser.java

```diff
@@ -0,0 +1,105 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package com.autotune.analyzer.utils;
+
+import com.autotune.analyzer.k8sObjects.Metric;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestEvalExParser {
+
+    List<String> objFunctionsList = new ArrayList<>(List.of(
+            "(( throughput / transaction_response_time) /  max_response_time) * 100",
+            "request_sum/request_count",
+            "(1.25 * request_count) - (1.5 * (request_sum / request_count)) - (0.25 * request_max)",
+            "((request_count / (request_sum / request_count)) / request_max) * 100"
+    ));
```

Comment:
Move these to a constants class which has all the constant characters

Reply:
Updated